Formula/inadyn.rb

```diff
@@ -0,0 +1,36 @@
+class Inadyn < Formula
+  desc "Dynamic DNS client with IPv4, IPv6, and SSL/TLS support"
+  homepage "http://troglobit.com/inadyn.html"
```

Comment:
This redirects to <http://troglobit.com/projects/inadyn/>. Would you mind updating?